code/Hieu_L/python/weather_api.py

```diff
@@ -0,0 +1,45 @@
+import requests
+#----------- v1 displays weather
+# zip_code = input('enter zip code: ')
+def lat_and_long(zip_code):
+    response = requests.get(f'http://api.openweathermap.org/geo/1.0/zip?zip={zip_code}&appid=884cfd64f3a52a3354c76c381207cf1e')
+    data = response.json()
+    return data
+
+def current_weather(zip):
+    lat = zip['lat']
+    lon = zip['lon']
+    response =  requests.get(f'https://api.openweathermap.org/data/2.5/weather?lat={lat}&lon={lon}&appid=884cfd64f3a52a3354c76c381207cf1e')
+    data = response.json()
+    return data
+
+def three_day_forecast(zip):
+    lat = zip['lat']
+    lon = zip['lon']
+    response =  requests.get(f'https://api.openweathermap.org/data/2.5/forecast?lat={lat}&lon={lon}&appid=884cfd64f3a52a3354c76c381207cf1e')
+    data = response.json()
+    return data
```

Comment:
You need to be more explicit with the data you want to return. Don't just return the entire JSON object, extract the information you want to display